packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/hooks/useAdminRolePermissionLayout/index.js

```diff
@@ -0,0 +1,23 @@
+import { useFetchClient } from '@strapi/helper-plugin';
+import { useQuery } from 'react-query';
+
+export const useAdminRolePermissionLayout = (id, queryOptions = {}) => {
```

Comment:
Can we write a description and idea about what this returns as `data`? I don't think it's clear and the API route naming to me, doesn't correlate with the hook name (rather it doesn't make it clearer lol)

Reply:
To be honest: I can't tell you what the response is about. This is basically the whole layout of the page (the tabs, sections) and the different permissions/ conditions that the BE knows about for plugins, collection/ single-types ... . I really don't know what all of this is in detail - happy to explore it as a follow-up.

Ok for you?

Reply:
Sure, i do think it's a bit of an issue we don't know, but if it needs more exploration then all chill ✨